chef_postgres/files/pg_hba.conf

```diff
@@ -72,30 +72,22 @@
 # listen on a non-local interface via the listen_addresses
 # configuration parameter, or via the -i or -h command line switches.
 
+# CAUTION: Configuring the system for local "trust" authentication
+# allows any local user to connect as any PostgreSQL user, including
+# the database superuser.  If you do not trust all your local users,
+# use another authentication method.
 
 
-
-# DO NOT DISABLE!
-# If you change this first entry you will need to make sure that the
-# database superuser can access the database using some other method.
-# Noninteractive access to all databases is required during automatic
-# maintenance (custom daily cronjobs, replication, and similar tasks).
-#
-# Database administrative login by Unix domain socket
-local   all             postgres                                trust
-
 # TYPE  DATABASE        USER            ADDRESS                 METHOD
 
 # "local" is for Unix domain socket connections only
-local   all             all                                     md5
+local   all             all                                     trust
 # IPv4 local connections:
-host    all             all             127.0.0.1/32            md5
-# IPv4 remote connections - change this line if you want to limit access to specific IP ranges.
```

Comment:
This should still be there `host    all             all             0.0.0.0/0               md5`

Reply:
done

Reply:
I might be blind, but I'm not seeing the 0.0.0.0 line.

Reply:
sorry my mistake i'm adding it to the pg_hba_conf.erb.   these files really aren't used so i'm not sure why we need to update them? I'll add it in there as well